src/Randomizer.SMZ3/Progression.cs

```diff
@@ -105,6 +105,8 @@ public Progression(IEnumerable<Item> items)
         public int ReserveTank { get; private set; }
         public int Rupees { get; private set; }
 
+        public List<TrickType> EnabledTricks { get; set; } = new () { TrickType.SinglePowerBombForMorphBombs };
```

Comment:
I would put this property on `Config` instead. I also wonder if it wouldn't be more straightforward and simpler to just make each trick a separate property (like the other stuff on `Config`) rather than an enum like this. Maybe a TrickConfig class?

Reply:
Yeah, my general thought was that these would originate from a config. Just hadn't gotten to the point of loading it yet. I can make it into its own config if you'd prefer, though. The properties would be the boolean values that people can toggle and whatnot, and for each one there would be a method that would be passed in in the progression object so that it can check what items the player has?

Reply:
Yeah, basically.

I envisioned it as something like a **TrickConfig** class accessible through the main **Config** instance, and some extension methods for checking against items (or honestly just properties on **Progression**, I don’t see why they used extension methods here).